python-threatexchange/threatexchange/exchanges/clients/techagainstterrorism/api.py

```diff
@@ -94,18 +103,17 @@ def _get(
         self,
         endpoint: t.Optional[str] = None,
         auth_token: t.Optional[str] = None,
-        full_url: t.Optional[str] = None,
+        **params,
```

Comment:
ignorable: `**params` is slightly dangerous because you can accidentally create slightly misnamed versions of enpoint and auth_tok if you aren't paying attention